### PR TITLE
 Updates for Director-free process changes, fix #30 

### LINKED ISSUE #30: Updates for Director-free process changes
The 2023 Process introduces [changes, in particular a Director-free approach](https://www.w3.org/2023/Process-20230612/#changes-2021) and thus, this useful document (which has six mentions of Director decisions or Director involvement) should be updated to take these into account.

### PR DESCRIPTION
**Pull request by svgeesus**
The 2023 Process introduces [changes, in particular a Director-free approach](https://www.w3.org/2023/Process-20230612/#changes-2021) and thus, this useful document (which has six mentions of Director decisions or Director involvement) has been updated to take these into account.

### CODE REVIEW

**Review by koalie — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -71,15 +72,17 @@ <h2>
 to the Recommendation track. That includes drafting the Deliverables section of a WG charter, but especially when publishing a FPWD. </p>
 
 <p>These are offered as  guidelines, not a checklist of required items in every proposal.
-Meeting all the guidelines will not guarantee Director approval of the charter or FPWD, nor will failing to meet some block approval. <em>The goal
-of this effort is to add more structure and predictability to Rec track decisions while allowing plenty of room for innovation by 
-WGs and judgment by the Director.</em></p>
+  
+Meeting all the guidelines will not guarantee the <a href="https://www.w3.org/2023/Process-20230612/#def-w3c-decision">W3C decision</a> for <a href="https://www.w3.org/2023/Process-20230612/#cfp">approval of the charter</a>, or <a href="https://www.w3.org/2023/Process-20230612/#maturity-stages">advancement to FPWD</a>, nor will failing to meet some block approval. <em>The goal
```
[the remark this answers is not in the document]
Comment (plehegar) [reply index.html:76]: This avoids hardcoding a specific version of the Process.

File: index.html [hunk 2]
```diff
@@ -134,12 +137,12 @@ <h2>
 ready for the W3C Recommendation Track. No single factor is decisive, and this document avoids <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC 2119</a> "MUST" and "SHOULD" terminology, and should not be a taken
 as a checklist of necessary or sufficient criteria.  Nevertheless, some criteria are
 noted as "strongly recommended", and, if not met, an explanation of why they don't apply in a particular situation would 
-facilitate the Director's decision.  Different cases will involve different combinations of these factors. In determining whether to 
-approve moving work to the Recommendation track, the Director may consider
+facilitate the <a href="https://www.w3.org/2023/Process-20230612/#decision-types">W3C decision</a>.  Different cases will involve different combinations of these factors. In determining whether to
```
Comment (plehegar) [inline index.html:140]:
```suggestion
facilitate the <a href="https://www.w3.org/Consortium/Process/#decision-types">W3C decision</a>.  Different cases will involve different combinations of these factors. In determining whether to 
```

**Review by plehegar — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -53,6 +53,7 @@ <h4>Nearby</h4>
       <h2>
         <a name="status" id="status">Status</a>
       </h2>
+      <p>October 2023: Document updated to account for <a href="https://www.w3.org/2023/Process-20230612/#changes-2021">changes in Process 2023</a>, in particular the Director-free changes.</p>
```
Comment (plehegar) [inline index.html:56]:
```suggestion
      <p>October 2023: Document updated to account for <a href="https://www.w3.org/Consortium/Process/#changes-2021">changes in Process 2023</a>, in particular the Director-free changes.</p>
```

File: index.html [hunk 4]
```diff
@@ -71,15 +72,17 @@ <h2>
 to the Recommendation track. That includes drafting the Deliverables section of a WG charter, but especially when publishing a FPWD. </p>
 
 <p>These are offered as  guidelines, not a checklist of required items in every proposal.
-Meeting all the guidelines will not guarantee Director approval of the charter or FPWD, nor will failing to meet some block approval. <em>The goal
-of this effort is to add more structure and predictability to Rec track decisions while allowing plenty of room for innovation by 
-WGs and judgment by the Director.</em></p>
+  
+Meeting all the guidelines will not guarantee the <a href="https://www.w3.org/2023/Process-20230612/#def-w3c-decision">W3C decision</a> for <a href="https://www.w3.org/Consortium/Process/#cfp">approval of the charter</a>, or <a href="https://www.w3.org/Consortium/Process/#maturity-stages">advancement to FPWD</a>, nor will failing to meet some block approval. <em>The goal
```
Comment (plehegar) [inline index.html:76]:
```suggestion
Meeting all the guidelines will not guarantee the <a href="https://www.w3.org/Consortium/Process/#def-w3c-decision">W3C decision</a> for <a href="https://www.w3.org/Consortium/Process/#cfp">approval of the charter</a>, or <a href="https://www.w3.org/Consortium/Process/#maturity-stages">advancement to FPWD</a>, nor will failing to meet some block approval. <em>The goal
```

File: index.html (hunk 3 above)
Comment (svgeesus) [reply index.html:56]: Since that one mentions changes in a particular dated version, it seems the dated link is appropriate there?

**Pull request merged**